src/observers/BaseStatusBarObserver.ts

```diff
@@ -0,0 +1,28 @@
+import { StatusBarItem } from 'vscode';
+import { BaseEvent } from '../common/loggingEvents';
+
+export abstract class BaseStatusBarItemObserver {

```

Comment:
>BaseStatusBarItemObserver [](start = 22, length = 25)

Any other class besides SignStatusBarObeserver.ts will use this?

Reply:
there will be a buildStatusBarObserver in next PR

---
In reply to: [363139713](https://github.com/docascode/vscode-docs-build/pull/10#discussion_r363139713) [](ancestors = 363139713)